### ISSUE TITLE
Some Accounts can't login

### ISSUE THREAD

**Issue by choell** [AHAAAAAAA/PokemonGo-Map#1514]
Some Accounts can't login via PTC:

2016-07-23 14:54:59,215 [   auth_ptc] [   INFO] PTC login for: xxx
2016-07-23 14:54:59,765 [   auth_ptc] [  ERROR] Could not retrieve token: Your username or password is incorrect. You have 1 attempts left before you will be locked out of your account for 15 minutes.
2016-07-23 14:54:59,765 [     search] [   INFO] Failed to login to Pokemon Go. Trying again.

Username and password are correct - login via website works

**Comment by ctluciani** [comment]
https://www.pokemon.com/us/pokemon-trainer-club/login shows this for me:

<img width="676" alt="5xmyfaz" src="https://cloud.githubusercontent.com/assets/7573372/17078030/136b9ee6-50a5-11e6-807b-8465d15aae64.png">

**Comment by diemer** [comment]
I'm having this on a number of python based Pokemon Go projects, and I think I may have tracked it down to having a % and a $ in my password. Gonna remove those and give it a shot (after this 15 minute ban is up). I'll report back

**Comment by diemer** [comment]
That was indeed the issue for me. Keep an eye out for special chars in passwords.

**Comment by gaMOTone** [comment]
Having the same issue. Username and password are definitely correct, and login works with app and PTC website. No special characters in my password.

**Comment by thedoofa** [comment]
Same here - no special characters either. Password is correct and can login via web.

**Comment by kedf** [comment]
same here too

**Comment by GhS88** [comment]
I am not sure if this will be helpful, but two of the accounts i am using stopped working, and it turned out that i needed to validate them first

**Comment by hemsleyk** [comment]
Yes, PTC accounts work immediately when created but deactivate after two days if not verified.
